### ISSUE TITLE
Add an iFrame-Name to Google Maps Element

### ISSUE THREAD

**Issue by Sebshn** [soflyy/oxygen-bugs-and-features#2354]
Hi there,

The Google Maps Feature is neat but please add a title to it to make it more accessible and increase Lighthouse scores.
Documentation: https://web.dev/frame-title/?utm_source=lighthouse&utm_medium=devtools

Which label is needed: title

Thank you.

![image](https://user-images.githubusercontent.com/48487853/129757283-360e3e68-77f0-4c19-b632-b277d3807d53.png)

Thanks and love.

**Comment by jamescrabb** [comment]
should this not be labelled as a bug?

**Comment by KittenCodes** [comment]
> should this not be labelled as a bug?

If something isn't working exactly as intended then it's a bug. This isn't the case here, so it's a feature request.

**Comment by MattHag** [comment]
Yes, please add this. Everything that Oxygen can easily handle for the user should be in there.
I hope this gets into the next Version as one of little fixes. 

As I aware that @Spellhammer is committed to improve his knowledge on accessibility and let that flow to Oxygen, I am sure this will be addressed.
